include/modem_key_mgmt.h

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
```

Comment:
2019

Reply:
I'll keep 2018 as this is derivative work from `nrf_inbuilt_key.h` that has already been released since 2018.